openhcl/underhill_attestation/src/igvm_attest/ak_cert.rs

```diff
@@ -4,31 +4,45 @@
 //! The module for `AK_CERT_REQUEST` request type that supports parsing the
 //! response.
 
-use crate::igvm_attest::Error;
 use crate::protocol::igvm_attest::get::IgvmAttestAkCertResponseHeader;
 use crate::protocol::igvm_attest::get::AK_CERT_RESPONSE_HEADER_VERSION;
+use thiserror::Error;
 use zerocopy::FromBytes;
 
+#[derive(Debug, Error)]
+pub enum AkCertError {
```

Comment:
The build is fine and the definition seems to be found.  I don't think this is required.

Reply:
hmm, it's actually used at
https://github.com/microsoft/openvmm/blob/7b007aaae8311069e1cc5c885a61f62ea0022dae/openhcl/underhill_core/src/emuplat/tpm.rs#L109
which we don't really enforce the type.
Can you try removing `pub` here and see if build still passes?

Reply:
It might be possible that these errors are implicitly exposed as part of re-export of `parse_response`. This would explain why the build passes with this change. We can at least add comment to explain the reason for this `pub` (if it's required).